ipwb/webui/index.html

```diff
@@ -20,8 +20,8 @@
 <div id="wrapper">
   <div>
     <h1><img src="./webui/logo.png" alt="ipwb" /></h1>
-    <form onsubmit="handleSubmit(); return false;">
-      <input type="text" placeholder="http://example.com" id="url" aria-label="Enter a URI" />
+    <form onsubmit="handleSubmit(); return false;" method="get" action="/memento/*/">
+      <input type="text" placeholder="http://example.com" id="url" name="url" aria-label="Enter a URI" />
```

Comment:
In future we might want to revisit this name if we make the search field more generic and accept queries in natural language like `example.com 3 months ago`.

Reply:
I agree. Fuzzy/free text queries would be a neat feature to have and I think would be novel as a search method for an archival replay system.